modules/navier_stokes/doc/content/source/linearfvbcs/LinearFVP1RadiationMarshakBC.md

```diff
@@ -0,0 +1,27 @@
+# LinearFVP1RadiationMarshakBC
```

Comment:
@GiudGiud what do you think, should this go here or the heat transfer module?

Reply:
Added to the heat transfer module.